test/test_view_ops.py

```diff
@@ -346,6 +346,14 @@ def compare_with_numpy(contiguous_input=True):
         self.assertEqual(a[5:].real, a.real[5:])
         self.assertEqual(a[5:].imag, a.imag[5:])
 
+    @onlyOnCPUAndCUDA
+    @dtypes(*torch.testing.get_all_complex_dtypes())
+    def test_conj_view(self, device, dtype) -> None:
```

Comment:
Should we add a NumPy interoperability test for calling `.numpy()` on a tensor whose conjugate bit is set?

We have a test suite for NumPy interop here:

https://github.com/pytorch/pytorch/blob/master/test/test_numpy_interop.py